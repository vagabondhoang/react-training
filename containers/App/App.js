import React from 'react';
import { BrowserRouter, Route, Switch } from 'react-router-dom';

import Home from '../HomePage/index';
import About from '../AboutPage/About';
import Users from '../UsersPage/Users';

const App = () => (
  <BrowserRouter>
    <Switch>
      <Route exact path="/" component={Home} />
      <Route path="/about" component={About} />
      <Route path="/users" component={Users} />
    </Switch>
  </BrowserRouter>
);

export default App;
