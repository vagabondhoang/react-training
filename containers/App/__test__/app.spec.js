import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import App from '../App';

Enzyme.configure({ adapter: new Adapter() });

describe('<App /> component', () => {
  it('should', () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
