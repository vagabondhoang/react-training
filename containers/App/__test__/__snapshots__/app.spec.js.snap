// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> component should 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/about"
    />
    <Route
      component={[Function]}
      path="/users"
    />
  </Switch>
</BrowserRouter>
`;
